lib/vagrant-openshift/action/push_openshift_images.rb

```diff
@@ -148,7 +148,7 @@ def call(env)
 
           cmd += %{
 # so we can call s2i
-export PATH=/data/src/github.com/openshift/source-to-image/_output/go/bin:/data/src/github.com/openshift/source-to-image/_output/local/go/bin:$PATH
+export PATH=/data/src/github.com/openshift/source-to-image/_output/go/bin:/data/src/github.com/openshift/source-to-image/_output/local/bin/linux/amd64:$PATH
```

Comment:
i think you changed the wrong one.  Today the binary is in _output/local/go/bin, so that needs to be in the path.

_output/go/bin appears to be dead and could be removed.